.github/workflows/ci.yml

```diff
@@ -22,3 +24,37 @@ jobs:
         run: |
           git-secrets --register-aws
           git-secrets --scan
+  doxygen:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          path: freertos
+      - name: Install Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Download tag dependency of coreMQTT-Agent
+        run: |
+          # We don't need to generate the coreMQTT docs, we only need the tag file.
+          # Therefore, we can just download it.
+          mkdir -p freertos/FreeRTOS-Plus/Source/Application-Protocols/coreMQTT-Agent/source/dependency/coreMQTT/docs/doxygen/output
```

Comment:
I would prefer that we regenerate the documentation for coreMQTT from whatever revision the sub module is currently at, since the docs on FreeRTOS.org do not appear to be versioned.

Reply:
That is certainly the right/ideal way to do it, however, currently, the coreMQTT-Agent has a direct dependency on the hosted version of coreMQTT doxygen (as can be seen [here](https://github.com/FreeRTOS/coreMQTT-Agent/blob/134b478a4862aa084575e3c37ab9fc86c04d3509/docs/doxygen/config.doxyfile#L2229)) and thus, it is required for the `mqtt.tag` to match the hosted documentation on freertos.org.

In the future, when the coreMQTT submodule is advanced past the LTS tag, we will have to find a better way to address the dependency.